app/assets/javascripts/components/story/StoryDescription.js

```diff
@@ -1,34 +1,42 @@
 import React from 'react';
-import Parser from 'html-react-parser'
-import memoize from 'memoizee';
+import AtWhoInput from 'components/jquery_wrappers/AtWhoInput';
+import DescriptionContent from 'components/description/DescriptionContent';
 
-import StoryLink from 'components/stories/StoryLink';
+class StoryDescription extends React.Component {
+  AtWhoInput() {
```

Comment:
Can we use a better name for this method? We should never call methods with PascalCase. What do you think about calling it `editDescription()`?